server/cmd/mmctl/commands/export.go

```diff
@@ -134,24 +135,36 @@ func init() {
 func exportCreateCmdF(c client.Client, command *cobra.Command, args []string) error {
 	data := make(map[string]string)
 
-	excludeAttachments, _ := command.Flags().GetBool("no-attachments")
-	if !excludeAttachments {
-		data["include_attachments"] = "true"
-	}
+	usersOnly, _ := command.Flags().GetBool("users-only")
+	if usersOnly {
+		data["users_only"] = "true"
 
-	excludeRolesAndSchemes, _ := command.Flags().GetBool("no-roles-and-schemes")
-	if !excludeRolesAndSchemes {
-		data["include_roles_and_schemes"] = "true"
-	}
+		// For users-only export, only profile pictures option is relevant
```

Comment:
Actually, I think we should error if other flags are used. They have no effect when using `users-only`.